src/main/resources/data.sql

```diff
@@ -393,7 +393,53 @@ INSERT INTO county (
 /**********************************
  * Begin Program-Counties.
  **********************************/
--- TODO
+INSERT INTO program_county (
```

Comment:
This is just some sample data to associate some counties to our sample programs (so you don't have to save something in the UI to be able to see counties on the map -- when the KML PR is merged).